lib/puppet/type/keystore_certificate.rb

```diff
@@ -0,0 +1,67 @@
+Puppet::Type.newtype(:keystore_certificate) do
+  desc 'adds a certificate and private key to a PKCS12 keystore'
+
+  ensurable
+
+  def self.title_patterns
+    [
+      [
+        /(.+):(.+)/m,
```

Comment:
Do we expect the possibility of multi-line resource titles?

Further  below (L13) do do we intend to map multi-line resource titles to an alias only?

Some elaboration on the choice of `/.../m` would be helpful to review this

Reply:
I should know this but what is a multi-line resource title?

Reply:
I did it this way to support `/path/to/keystore:alias` and `alias` as titles.

Reply:
I'm reading at https://ruby-doc.org/core-2.4.1/Regexp.html#EXTENDED

"/pat/m - Treat a newline as a character matched by ."

How does it relate to `/path/to/keystore:alias` ? There is some problem with the `/` characters in the path?

Reply:
I didn't much think about it. I copied this pattern from the `truststore_certificate` we had built previously, and that originates from https://github.com/theforeman/puppet-certs/pull/312